Translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [vodle/user interface](https://hosted.weblate.org/projects/vodle/user-interface/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/vodle/user-interface/horizontal-auto.svg)

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (591 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/pik-gane/vodle?pullRequest=281) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/pik-gane/vodle?pullRequest=281) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/pik-gane/vodle?pullRequest=281) it.</sub>

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (591 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (591 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

mensch72 committed:
Translated using Weblate (Tamil)

Currently translated at 99.8% (590 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 99.8% (590 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 99.1% (586 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 62.9% (372 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 65.3% (386 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 69.8% (413 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 74.9% (443 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 96.6% (571 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 96.7% (572 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

Commit:
Translated using Weblate (Tamil)

Currently translated at 100.0% (591 of 591 strings)

Translation: vodle/user interface
Translate-URL: https://hosted.weblate.org/projects/vodle/user-interface/ta/

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'

weblate committed:
Merge remote-tracking branch 'origin/main'
